pkg/params/run.go

```diff
@@ -70,15 +69,6 @@ func (r *Run) GetConfigFromConfigMap(ctx context.Context) error {
 		r.Info.Pac.RemoteTasks = StringToBool(remoteTask)
 	}
 
-	if timeout, ok := cfg.Data["default-pipelinerun-timeout"]; ok {
-		parsedTimeout, err := time.ParseDuration(timeout)
-		if err != nil {
-			r.Clients.Log.Errorf("failed to parse default-pipelinerun-timeout: %s", cfg.Data["default-pipelinerun-timeout"])
-		} else {
-			r.Info.Pac.DefaultPipelineRunTimeout = &parsedTimeout
```

Comment:
Could you also remove `DefaultPipelineRunTimeout` from the struct ?
so we will know if its being used some where .

Reply:
Done removed from all places